.gitignore

```diff
@@ -0,0 +1,17 @@
+.env
+.DS_Store
+venv/
+*.pyc
+__pycache__/
+instance/
+
+# Recommended exclusions by Flask
+# .pytest_cache/
+# .coverage
+# htmlcov/
+# dist/
+# build/
+# *.egg-info/
+
```

Comment:
does `egg-info` need to be ignored as well?

Reply:
Yes. I think it probably should be. We can remove on future PR.

Reply:
Don't believe there's any compromising info in there, just best practice to have it not in the repo.

Reply:
Update: Removed & updated gitignore.